moped-editor/src/views/projects/projectView/ProjectComponents.js

```diff
@@ -63,7 +69,9 @@ const ProjectComponents = () => {
   const { projectId } = useParams();
   const classes = useStyles();
 
-  const [selectedComp, setSelectedComp] = useState(0);
+  const [selectedProjectComponent, setSelectedProjectComponent] = useState(
```

Comment:
in addition to renaming this state var, it's now getting passed down the tree and supplies the initial map edit state